clientgui/MainDocument.cpp

```diff
@@ -1719,7 +1719,7 @@ void CMainDocument::KillGraphicsApp(int pid) {
         argv[4] = thePIDbuf;
         argv[5] = 0;
 
-        iRetVal = run_program(
+        run_program(
```

Comment:
I'd here add proper error handling in case there was unsuccessful try to run this application (if we can't properly deal with this situation, we should at least write a message to the log file). Ignoring errors silently is not nice.

Reply:
Again, this is not my code. Rom Walton wrote it many years ago. While I agree with you, it is well beyond the scope of this PR and not something I am willing to do. If you feel strongly, I suggest you create a separate issue.